docs/oss/links.md

```diff
@@ -12,6 +12,8 @@ sidebar_position: 98
 
```

Comment:
Should we migrate the foss-example?

Reply:
No, we should delete / deprecate it. All templates are now referenced directly to EF sources or to examples in TractusX. I removed the link.